rust_src/src/files.rs

```diff
@@ -0,0 +1,111 @@
+extern crate libc;
+extern crate rand;
```

Comment:
I think this should be on the crate root, and in this file add the respective `use` declarations.